### PR TITLE
Fix Android icons

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

File: gulpfile.js [hunk 1]
```diff
@@ -379,8 +379,20 @@ function dist_src() {
...
+    const cssSources = [
+        './node_modules/**/*.min.css',
+    ];
+
+    if (isAndroid) {
+        cssSources.push("./node_modules/**/*.woff2");
+        cssSources.push("./node_modules/**/*.ttf");
+    }
+
+    return gulp
```
Comment (KarateBrot) [inline gulpfile.js:394]:
```suggestion
  return gulp
```

Comment (haslinghuis) [reply gulpfile.js:394]: 4 space indentation is correct.

Comment (KarateBrot) [reply gulpfile.js:394]: Then the two lines below miss 4 intendations but no worries 😃

**Pull request merged**